PowerEditor/src/NppIO.cpp

```diff
@@ -2246,6 +2246,19 @@ bool Notepad_plus::loadSession(Session & session, bool isSnapshotMode, bool shou
 			continue;	//skip session files, not supporting recursive sessions or embedded workspace files
 		}
 
+		if (session._subViewFiles[k]._hasClone)
+		{
+			// it appears that the user had this document cloned when they last saved the session
+			// if it's already open in the main edit view, we'll just clone it over from there.
+			BufferID copyInMainView = _mainDocTab.findBufferByName(pFn);
+			if (copyInMainView != BUFFER_INVALID)
+			{
+				loadBufferIntoView(copyInMainView, SUB_VIEW);
+				++k;
+				continue;
+			}
+		}

```

Comment:
Detect here if the same file path can be found in main view - since user cannot open the same file in 2 views at the same time, except by mean of cloning.
So 
```cpp
			BufferID copyInMainView = _mainDocTab.findBufferByName(pFn);
			if (copyInMainView != BUFFER_INVALID)
			{
				loadBufferIntoView(copyInMainView, SUB_VIEW);
				++k;
				continue;
			}
```
is all we need to fix the problem.